package.json

```diff
@@ -1,6 +1,6 @@
 {
 	"name": "@getflywheel/local-components",
-	"version": "13.2.2",
+	"version": "13.2.3",
```

Comment:
No need to bump the `version` number here -- This gets bumped once we cut an official release. @flycrum could speak more to this process.

Reply:
This is a great point, @veryspry! We'll take care of the version number after the PR is merged/closed, so yes, it'd be best if this was removed. Thanks.